packages/cli/src/commands/init/init.ts

```diff
@@ -268,6 +270,24 @@ function createTemplateUri(options: Options, version: string): string {
   return options.template || `react-native@${version}`;
 }
 
+function createDefaultConfigFile(directory: string) {
+  const content = `module.exports = {
+  project: {
+    ios: {
+      automaticPodsInstallation: true,
+    },
+  },
+};
+`;
```

Comment:
Hm, I'm wondering if this shouldn't live in the template in Core? WDYT?

Reply:
I was thinking about it, but IMO it would bring a lot of confusion for people using Upgrade Helper, as for existing apps it's not something you'd like to have in the project. This solution makes it totally optional for already existing projects